lib/prelude.nat

```diff
@@ -43,3 +43,25 @@ unit °C = celsius
 
 conv x:K ⇒ °C = (x / K)°C - 273.15°C
 conv x:°C ⇒ K = (x/°C + 273.15)K
+
+unit joule = kg * (m / s)^2
+unit J = joule
+
+unit angstrom = 100 pm
+unit ångström = angstrom
+unit Å = angstrom
+
+unit micron = μm
+
+c = 299792458
```

Comment:
Shouldn't this be

```suggestion
c = 299792458 m/s
```
?

Reply:
Forgot to change this before committing. Thanks for the catch 😅